### ISSUE TITLE
Possible to use it with partners?

### ISSUE THREAD

**Issue by Vulpecula-nl** [TheAxeDude/ZiniGo#10]
There are a few partners of Zinio which use there own layout but with the backend of zinio. Maybe is it useful to add the url also to the config file.

**Comment by TheAxeDude** [comment]
We can definitely try. Do you have an example of one of the partners/their
website?

**Comment by Vulpecula-nl** [comment]
yes, I have. I view a magazine online on https://tijdschrift.nl/ and see it's using Zinio

**Comment by TheAxeDude** [comment]
Thanks! I'll dig around and see what I can see :)

**Comment by Vulpecula-nl** [comment]
Found the url to get all the pages of a magazine.

**Comment by TheAxeDude** [comment]
Can you paste the URL here? I'm having issues signing up for an account with tijdschrift.

**Comment by Vulpecula-nl** [comment]
https://dpgmediareader.zinioapps.com/api/newsstand/newsstands/3219/issues/491255/content/pages?format=svg&application_id=11221&css_content=true&user_id=xxx

Remove my user_id. In the header there is also an authorization parameter
